cognite/powerops/custom_modules/power_model_v1/data_models/containers/SHOPTimeSeries.container.yaml

```diff
@@ -0,0 +1,51 @@
+space: '{{powerops_models}}'
+externalId: SHOPTimeSeries
+name: SHOPTimeSeries
```

Comment:
Yes, this is what I thought of. Worth discussing with someone else if this is a good idea, has some important drawbacks, or can be solved in another way? (Maybe this will give us too many instances, maybe there is/will be a way to extend the TimeSeries type directly/...)